integration-tests/reactive-messaging-kafka/pom.xml

```diff
@@ -175,6 +175,13 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
I'm not sure I like this fix. Which dependency does bring the old version?

Reply:
Reactive messaging. I don't like it too but is there any other way around it?

Reply:
Please don't disable _all_ enforcer rules like this! I'll take a look and see if I can help.

Reply:
IIUC, MP Health API 3.1 shall be used? If so, I strongly suggest to apply this:
```diff
diff --git a/integration-tests/reactive-messaging-kafka/pom.xml b/integration-tests/reactive-messaging-kafka/pom.xml
index 184230787f..3ca5928173 100644
--- a/integration-tests/reactive-messaging-kafka/pom.xml
+++ b/integration-tests/reactive-messaging-kafka/pom.xml
@@ -51,6 +51,13 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-reactive-messaging-kafka</artifactId>
+            <exclusions>
+                <!-- avoid convergence error; use health-api version from quarkus-smallrye-health -->
+                <exclusion>
+                    <groupId>org.eclipse.microprofile.health</groupId>
+                    <artifactId>microprofile-health-api</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>


@@ -175,13 +182,6 @@

     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <configuration>
-                    <skip>true</skip>
-                </configuration>
-            </plugin>
             <plugin>
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-maven-plugin</artifactId>
```

Reply:
I'll try, might work as this wasn't a breaking release. Of course this will need to be reverted in every case once RM is updated too.

Reply:
FYI, this exclusion will be removed by #18334